fleeto_backend/app/models/trip_request.rb

```diff
@@ -4,15 +4,15 @@ class TripRequest < ApplicationRecord
   
   has_one :trip
   
-  validates :from_lat, :from_long, presence: true
+  validates :from_latitude, :from_longitude, presence: true
 
   validate :customer_is_not_in_a_trip
 
   PENDING, CANCELED, ACCEPTED, EXPIERED = STATUSES = %w(pending canceled accepted expired)
   enum_string :status, STATUSES
 
   # TODO this should be configurable from the admin
-  DISTANCE_IN_KM_TO_SEARCH_DRIVERS_WITHIN = 5
+  DISTANCE_IN_KM_TO_SEARCH_DRIVERS_WITHIN = 1000
```

Comment:
We need to track all these settings so that we allow admin to control their values from a web interface.